.etc/example_config.toml

```diff
@@ -0,0 +1,12 @@
+host = "0.0.0.0"  # IP or hostname where the server will bind
+port = 25565  # Server port (0-65535)
+motd = ["Welcome to the server!", "Have a great time!", "Adventure awaits!"]  # Message of the day
+max_players = 100  # Maximum number of players
+network_tick_rate = 20  # Network updates per second per user
+world = "World"  # The name of the world to load
```

Comment:
Wasn't the previous value for this `world = "world"`?

Reply:
yes, but it doesn't really matter

Reply:
> yes, but it doesn't really matter

Yeah I guess it's not used yet still think it should be the same as it was before thougt